### DISCUSSION TITLE
Replacing the OP Amp

### DISCUSSION THREAD

**Discussion by axizhe** [ozel/DIY_particle_detector#46] [Q&A] [answered]
Hello, my name is Sergio Ruppel, from Argentina. Currently building the electron detector. 

Im having real trouble to find the right op amp, since it's not even imported right now, i used the cheaper TL082, as it seems it's only short in the Slew rate compared to the recomended one, but did my own PCB design as small as possible and im not getting any good measurement.. testing with Potassium from bananas, but about to buy KCL today or tomorrow. Could it be that the Op Amp is actually bad for that purporse? 
I may be able to replace it with a NE5532.. 

I'm mising something more important? 
Thanks, Sergio

**Comment by ozel** [comment, marked as answer]
Hi Sergio,

that's really annoying if you can not get all the parts.
But the TL082 should still work. I recommend the TL072 as an alternative to the more expensive but nicely low-noise TLE2072. The differences between TL07 and TL08 series are not so significant. A few pulses may be hidden in noise, but if you still use only 4 diodes it should be ok (more diodes in parallel = more noise).
See here for some component alternatives:
https://kitspace.org/boards/github.com/ozel/diy_particle_detector/electron-detector/

The slew rate and bandwidth of teh op amp do not really matter. Important are low current noise and low voltage noise figures, high-impedance and low-capacitance JFET or BiFET style inputs.

I would highly advise that you build my original detector design exactly as it is first before making your own. 
As long as you have not made the original design work, it is impossible for me to help you with your own design.
My design will work on a vero board if you solder carefully and put all parts close together.
But there are a few other key points such as using only a 9V battery for power and having it all in a fully light-tight metal box.
Did you see the troubleshooting list? Every single point counts.
https://github.com/ozel/DIY_particle_detector/wiki/Troubleshooting

You can forget about measuring bananas. No cheap detector is able to do that. The youtube videos claiming that are all wrong and simply measure radon and cosmic radiation! 
If you put 10 grams of KCL in front of the 4 diodes, you will see about 1.3 pulses **per minute**.
For more details, see chapter 3.3 here: https://www.mdpi.com/1424-8220/19/19/4264/htm
So take at least 100 g of KCL if you can get it from a pharmacy or as diet salt from an organic/alternative food store.
A banana contains less than 1 g of potassium, and most of the radiation is shielded by the other matter that the banana is made of...

I recommend getting some vintage uranium glass from a local flea market or eBay if you can. It is much easier to measure.
